kgforge/specializations/resolvers/ontology_resolver.py

```diff
@@ -48,25 +50,23 @@ def _resolve(self, text: Union[str, List[str]], target: Optional[str], type: Opt
         if type:
             first_filters = f"{first_filters} ; a {type}"
 
+        properties = ['label', 'notation', 'prefLabel', 'altLabel']
         if strategy == strategy.EXACT_MATCH:
-            label_filter = f" FILTER (?label = \"{text}\")"
-            notation_filter = f" FILTER (?notation = \"{text}\")"
-            prefLabel_filter = f" FILTER (?prefLabel = \"{text}\")"
-            altLabel_filter = f" FILTER (?altLabel = \"{text}\")"
-            limit = 1
+            regex = False
+            case_insensitive = False
```

Comment:
Did we lost the limit = 1 for EXACT_MATCH or is it some default value somewhere?